src/app/features/auth/components/login-form/login-form.component.ts

```diff
@@ -0,0 +1,65 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AbstractControl, FormGroup, NonNullableFormBuilder, ReactiveFormsModule, Validators } from "@angular/forms";
+import { Credentials } from "../../../../core/auth/types/credentials";
+
+@Component({
+  selector: 'login-form',
+  standalone: true,
+  imports: [CommonModule, ReactiveFormsModule],
+  templateUrl: './login-form.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LoginFormComponent {
+
+  @Input() public submitted: boolean | null;
+  @Output() public readonly submitEmitter = new EventEmitter<{ credentials: Credentials; isRemember: boolean }>();
+  @Output() public readonly markFieldsSubmittedEmitter = new EventEmitter<boolean>();
+
+  public form: FormGroup = this._fb.group({
+    email: [
+      '',
+      {
+        validators: [Validators.required, Validators.email]
+      }
+    ],
+    password: [
+      '',
+      {
+        validators: [Validators.required, Validators.minLength(6)]
+      }
+    ],
+    rememberMe: [false]
+  })
```

Comment:
Make a habit of oneliners please. Everything I see could be reduced in size. For example this block could be:
```typescript
  public form: FormGroup = this._fb.group({
    email: ['', [Validators.required, Validators.email]],
    password: ['', [Validators.required, Validators.minLength(6)]],
    rememberMe: false
  })
```

This applies to everything: objects, arrow functions, arrays, etc.